contrib/connector/rbd/rbd.go

```diff
@@ -41,6 +42,23 @@ func init() {
 	connector.RegisterConnector(connector.RbdDriver, &RBD{})
 }
 
+func convertToStrList(in interface{}) ([]string, error) {
+	var out []string
+	switch in.(type) {
+	case []string:
+		out = in.([]string)
+	case []interface{}:
```

Comment:
Have you tested it here? The type of in is interface{}, but in.(type) can be []interface{}?